dist/UserLeap-Kit.iife.js

```diff
@@ -0,0 +1,782 @@
+var UserLeapKit = (function (exports) {
```

Comment:
totally your call, but we find it best to commit all src code, then in once approved, do another commit to build the files. while this doesn't guarantee that you will build the file before committing it as part of step 2, it reminds you to.  this applies to both `UserLeap-Kit.iife.js` and `UserLeap-Kit.common.js` above